api/v1alpha1/vspheredistributednetwork_types.go

```diff
@@ -123,12 +123,66 @@ type VlanSpec struct {
 	PrivateVlanID *int32 `json:"privateVlanID,omitempty"`
 }
 
+// MacLimitPolicyType represents the policy type to be used when the MAC address limit is exceeded.
+type MacLimitPolicyType string
+
+const (
+	// MacLimitPolicyAllow indicates that new MAC addresses should still be allowed when the limit is exceeded.
+	MacLimitPolicyAllow MacLimitPolicyType = "allow"
+	// MacLimitPolicyDrop indicates that new MAC addresses should be dropped when the limit is exceeded.
+	MacLimitPolicyDrop MacLimitPolicyType = "drop"
+)
+
+// MacLearningPolicy represents the MAC learning policy configuration.
+type MacLearningPolicy struct {
+	// Enabled indicates whether MAC learning is enabled.
+	Enabled bool `json:"enabled,omitempty"`
```

Comment:
Should this be 'required' or 'optional'? please either remove `omitempty` and add `// +required`, or add `// +optional`